shell/platform/linux/public/flutter_linux/fl_view.h

```diff
@@ -16,7 +16,7 @@
 
 G_BEGIN_DECLS
 
-G_DECLARE_FINAL_TYPE(FlView, fl_view, FL, VIEW, GtkContainer)
+G_DECLARE_FINAL_TYPE(FlView, fl_view, FL, VIEW, GtkBox)
```

Comment:
While this is changing the public API, I don't think this causes any ongoing issues - the class couldn't be derived and it only accessed via pointers and functions. There is no reason for an app to be looking inside this widget, and since it is a GtkBox it is still a GtkContainer.